src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,184 @@
+package core.basesyntax.service;
+
+import static core.basesyntax.db.Storage.people;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static List<User> users = new ArrayList<>();
+    private final StorageDao storageDao = new StorageDaoImpl();
+    private final RegistrationServiceImpl registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void beforeAll() {
+        users.add(new User("potrap", "qwerty123", 20));
+        users.add(new User("maksym", "jen478fs", 19));
+        users.add(new User("obama", "123456789", 65));
+        users.add(new User("baiden", "12365t89", 75));
+        users.add(new User("hellomoto", "hellooo123", 49));
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        people.clear();
+    }
+
+    @Test
+    void registration_nullAge_notOk() {
+        try {
+            registrationService.register(new User("potrap", "qwerty123", null));
+        } catch (RuntimeException e) {
+            return;
+        }
```

Comment:
```suggestion
        try {
            registrationService.register(new User("potrap", "qwerty123", null));
        } catch (RuntimeException e) {
            return;
        }
```
use `assertThrows` instead of try catch